clang/lib/APINotes/APINotesYAMLCompiler.cpp

```diff
@@ -793,6 +827,16 @@ class YAMLConverter {
                          SwiftVersion);
   }
 
+  template <typename T>
+  void convertVariable(const T &Entity, VariableInfo &VI) {
```

Comment:
I think that `computeVariableInfo` seems better - we are "hydrating" the variable information from the entity.

Reply:
The existing methods that do the same thing for classes, methods, etc. are called `convertX` without the `Info` suffix. I think it's better keep the naming consistent.